solutions/02-useLazyLoadQuery/src/App.tsx

```diff
@@ -9,8 +9,8 @@ import { AppQuery } from './__generated__/AppQuery.graphql';
 const App = () => {
   const response = useLazyLoadQuery<AppQuery>(
     graphql`
-      query AppQuery {
-        posts(first: 10) {
+      query AppQuery ($first: Number) {
```

Comment:
Number does not exist

![image](https://user-images.githubusercontent.com/2005841/96450163-3a1f0e80-11ec-11eb-9f0e-6b5e86349337.png)